### ISSUE TITLE
RealmSwift Carthage for Xcode 8 and Swift 3?

### ISSUE THREAD

**Issue by xspyhack** [realm/realm-cocoa#4099]
The existed frameworks be created by an older version of the compiler.

**Comment by kishikawakatsumi** [comment]
@xspyhack Xcode 8 and Swift 3 compatible RealmSwift will be released soon. For now, you should specify `--no-use-binaries` option to avoid Carthage download pre-built binaries. Also specify `master` branch, and toolchain using Swift 3. 

So a Cartfile is:

```
github "realm/realm-cocoa" "master"
```

carthage command and options are:

```
TOOLCHAINS=com.apple.dt.toolchain.Swift_3_0 carthage update --no-use-binaries
```

**Comment by liuxuan30** [comment]
any precise release date?

**Comment by kishikawakatsumi** [comment]
@liuxuan30 We've released `v1.1.0` already. https://github.com/realm/realm-cocoa/releases/tag/v1.1.0

**Comment by liuxuan30** [comment]
is it for swift 3.0? I was thinking it's not
